docs/source/algorithms/XrayAbsorptionCorrection-v1.rst

```diff
@@ -0,0 +1,75 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm calculates the correction factors due to the absorption of Xrays 
+by sample and applies it to Input Workspace. This is done by determining path of xray from
+muon in sample to detector and using the calculated distance in sample to calculate 
```

Comment:
I'm not sure what distance it is using. You could put:

muon in the sample to the detector. It calculates the distance between ???? and the sample to calculate the correction function using

then you should use the maths formatting for the equation